### PR TITLE
Add a bunch of documentation, consolidate what was already there.

### PR DESCRIPTION
**Pull request by pixelbandito**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,24 @@
 
 [![NPM](https://img.shields.io/npm/v/@cision/rover-ui.svg)](https://www.npmjs.com/package/@cision/rover-ui)
 
+RoverUI is a collection of UI components originally built for and by Cision. These components should help the Cision team build applications that look hot, in React, quickly.
+
 ## Install
 
 ```bash
+# yarn
+yarn rover-ui
```
Comment (chelshaw) [inline README.md]: this doesn't need `install`!? TIL

File: README.md [hunk 2]
```diff
@@ -24,7 +36,42 @@ class Example extends Component {
...
+
+### What to avoid in RoverUI
+
+It's _not_ a repository for all shared components. Don't try to add components that deal specifically with business logic and concepts here. Don't try to add components that handle whole layout areas.
+
+## Run locally
+
+0. Read the [Code Guidelines](./CODE_GUIDELINES.md)
+1. Download the repository
+1. From the root, run `yarn` to install dependencies.
```
Comment (chelshaw) [inline README.md]: you don't wanna update these numbers do you

File: README.md (hunk 1 above)
Comment (mdespuits) [reply README.md]: Wait, seriously? TIL too!

File: README.md [hunk 3]
```diff
@@ -39,12 +86,33 @@ Loaded templates: _templates
...
        added: src/components/Dropdown/README.md
        added: src/components/Dropdown/story.js
-      inject: src/stories.js
+      inject: src/stories/index.js
        added: src/components/Dropdown/test.js
 ```
 
 At any time you can see what generators we have configured using `hygen help`.
 
+### Customizing the theme with a custom build
```
Comment (chelshaw) [inline README.md]: Seems like this section should be after the CSS-in-JS section since it builds off that info

File: README.md (hunk 2 above)
Comment (mdespuits) [reply README.md]: Technically they could be all zeros and I think it would still work

File: README.md (hunk 1 above)
Comment (pixelbandito) [reply README.md]: Or, more likely, I typed it wrong.

File: README.md (hunk 2 above)
Comment (pixelbandito) [reply README.md]: I shall update them for prosperity.

File: README.md (hunk 3 above)
Comment (pixelbandito) [reply README.md]: I meant for this to be a preface to both css in js and modules sections. I added a little more to clarify it.

**Comment by rheckart** [comment]
Looks good

**Comment by pixelbandito** [comment]
@chelshaw Would you mind approving now that I've changed it?

**Pull request merged**